content/resources/operator/_index.md

```diff
@@ -112,7 +113,19 @@ One of the goals when providing an external reference is to figure out the level
 
 #### Creating Valuable Data Descriptions
 
-It's important to create clear and concise descriptions for each property in a dictionary.  The descriptions should be understandable by someone who is not familiar with a particular domain.  When available, including the unit of measure in parenthesis at the end of the description would be helpful in cases where the unit of measure is not included elsewhere in the description. When a clear description is not readily available, it's recommended that an external vocabulary such as NCIt be used as they offer definitions for terms from a plethora of domains.
+It's important to create clear and concise descriptions for each property in a dictionary. The descriptions should be understandable by someone who is not familiar with a particular domain. When available, including the unit of measure in parenthesis at the end of the description would be helpful in cases where the unit of measure is not included elsewhere in the description. When a clear description is not readily available, it's recommended that an external vocabulary such as NCIt be used as they offer definitions for terms from a plethora of domains.
+
+#### Selecting the Property Type
+
+The Gen3 model allows for a selection of property types that can be used to specify or limit values submitted for each property.
+The available types follow the [jsonschema basic types](https://cswr.github.io/JsonSchema/spec/basic_types/):
```

Comment:
```suggestion
The available types are [jsonschema types](https://cswr.github.io/JsonSchema/spec/introduction/):
```

Reply:
changing the link because array, object, null are not listed on the "basic types" page